### PR TITLE
fix: msp receiver emulator

### PR DESCRIPTION
**Pull request by chmelevskij**
Fixes MSP receiver window

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by SteveCEvans** [comment]
Now working for me.

image

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]
Good work!

**Pull request merged**
